Codidact.Authentication.sln

```diff
@@ -1,21 +1,23 @@
 ﻿
```

Comment:
Are the changes here intentional or not? This looks like something automatically generated. Should it be changed like this or shouldn't it? Does it have any negative or positive consequences if changed?

Reply:
These are automatically generated Visual Studio changes.
This happened because I added a new Project (WebApp.Tests) so it went ahead for some reason and changed all the GUIDs and added a new configuration for that project.

It doesn't have any negative consequences but for the positive this is just how Visual Studio is.